drivers/LVHDSR.py

```diff
@@ -1365,8 +1365,6 @@ def _kickGC(self):
                     if e.code != errno.ETIMEDOUT:
                         raise
                     util.SMlog('failed to abort the GC')
-                finally:
```

Comment:
The 

```
finally: 
    return

```
caused the running GC to be stopped but didn't start a new one. There probably should be a return in the case of an exception however